deploy/kubernetes/README.md

```diff
@@ -0,0 +1,15 @@
+Step 1 (Create Credentials):
+Create Service Account Credential JSON on GCP:
+    - TODO: Add detailed steps on how to do this
+    - Requires both Compute Owner and Cloud Project Owner permissions
```

Comment:
That's pretty broad permissions, there are no reduced scopes for just pd?

Reply:
theres a "compute-admin" and a "compute-storage-admin" scope. However when I tried those for some reason there were not enough permissions to Attach. Everything else was fine, just not Attach. I will open an issue on this repo to revisit that and either it will work, or I will open a bug against the compute scopes because I think "Attach" should definitely be covered under "compute-admin"

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/3